src/pymedphys/logfile/_level1/logfileindex.py

```diff
@@ -96,7 +96,7 @@ def create_index_entry(new_filepath, delivery_details: OISDeliveryDetails,
             **attr.asdict(delivery_details)
         },
         'logfile_header': {
-            header._asdict()
+            **header._asdict()
```

Comment:
This was a bug introduced due to not having testing on the logfile portion of pymedphys